jumpscale/clients/taiga/taiga.py

```diff
@@ -217,3 +217,190 @@ def _get_project_status(project_statuses, status):
+        for project in projects:
+            SINGLE_PROJECT_TEMPLATE = dict()
+            SINGLE_PROJECT_TEMPLATE["name"] = project.name
+            SINGLE_PROJECT_TEMPLATE["issues"] = project.list_issues()
+            ALL_PROJECTS_TEMPLATES.append(SINGLE_PROJECT_TEMPLATE)
+        for project in ALL_PROJECTS_TEMPLATES:
+            text += f"##### {project.get('name')} \n"
+            for issue in project.get("issues"):
+                text += f"- {issue.subject} \n"
+
+        j.sals.fs.write_file(path=path, data=text)
+
+    def export_issues_per_user(self, user_name=None, path="/tmp/issues_per_user.md"):
+        """Export all the issues per specif user in a markdown file, if no user given
+        the method export issues for the current logged in user
+        
+        Args:
+            path (str): The path of exported markdown file.
+            user_name (str): The name of the user we want to get his issues.
+            
+        """
+        if user_name:
+            selected_user_name = user_name
+        else:
+            selected_user_name = self.username
+        user_id = self._get_user_id(selected_user_name)
+        text = f"""
+### {selected_user_name} Issues \n
+"""
+        issues = self.list_all_issues(user_id=user_id)
+        ALL_ISSUES_TEMPLATES = self.__get_issues_required_data(issues)
+        text = self.__render_issues_with_details(text=text, issues=ALL_ISSUES_TEMPLATES)
+        j.sals.fs.write_file(path=path, data=text)
+
+    def __get_stories_required_data(self):
+        """Get the required data from user stories to be used later in export(render)
+
+        Returns:
+            List: List of stories required details to be used in export(render).
+        """
+        All_STORIES_TEMPLATE = list()
+        users_stories = self.list_all_user_stories()
+        for story in users_stories:
+            SINGLE_STORY_TEMPLATE = dict()
+            SINGLE_STORY_TEMPLATE["subject"] = story.subject
+            if story.assigned_to_extra_info:
+                SINGLE_STORY_TEMPLATE["assigned"] = story.assigned_to_extra_info.get("full_name_display", "unassigned")
+                SINGLE_STORY_TEMPLATE["is_active"] = story.assigned_to_extra_info.get("is_active", "unknown")
+            tasks = list()
+            for task in story.list_tasks():
+                tasks.append(task.subject)
+            SINGLE_STORY_TEMPLATE["tasks"] = tasks
+            All_STORIES_TEMPLATE.append(SINGLE_STORY_TEMPLATE)
+        return All_STORIES_TEMPLATE
+
+    def export_all_user_stories(self, path="/tmp/all_stories.md"):
+        """Export all the user stories in a markdown file
+        
+        Args:
+            path (str): The path of exported markdown file.
+            
+        """
+        text = f"""
+### All User Stories \n
+"""
+        All_STORIES_TEMPLATE = list()
+        All_STORIES_TEMPLATE = self.__get_stories_required_data()
+
+        for story in All_STORIES_TEMPLATE:
```

Comment:
I think if we are going to use markdown, it's better to use a template (jinja template for example) instead of doing it like this.

Reply:
you are right,  it is much better to use jinja template but since the markdown is simple we are not in a bad need of it, if the markdown got more complex we have to use jinja template.

Reply:
we are just listing here